ci-operator/jobs/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main-presubmits.yaml

```diff
@@ -30323,6 +30411,94 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )(rosa_hcp-4.18-nightly-x86-node-density-cni-24nodes|remaining-required),?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I wonder why this presubmit has `always_run: true` when in the actual test it is set to false?

Reply:
yeah, that is weird.